### DISCUSSION TITLE
Should we release 0.68.8?

### DISCUSSION THREAD

**Discussion by itsashis4u** [reactwg/react-native-releases#80] [Patches]
Current: 0.68.7
Similar to **v0.69.12** where we fixed a crash related to FrameBasedAnimation on OnePlus and OPPO devices
Here is the PR to be cherry picked: https://github.com/facebook/react-native/pull/37487

**Comment by cortinico** [comment]
Closing this as we won't be releasing further patches for 0.68 as it's outside of the release support window: https://github.com/reactwg/react-native-releases#releases-support-policy

**Reply by itsashis4u** [reply]
Other than building react native from source, is there any other way to patch 0.68 with the OnePlus Frame animation crash fix?
@cortinico ?

**Reply by cortinico** [reply]
The recommended solution is to update to 69 or newer versions

**Reply by cipolleschi** [reply]
As @cortinico said, the suggested solution is to update to one of the versions in the supported window. Those versions are 0.70, 0.71 and 0.72, right now. Link to the [Support Policy](https://github.com/reactwg/react-native-releases#releases-support-policy)

**Reply by itsashis4u** [reply]
Got it. Thanks @cortinico  @cipolleschi
